scripts/install

```diff
@@ -221,6 +211,13 @@ ynh_script_progression --message="Starting a systemd service..." --weight=1
 # Start a systemd service
 ynh_systemd_action --service_name=$app --action="start"
 
+#=================================================
+# SEND A README FOR THE ADMIN
+#=================================================
+ynh_script_progression --message="Sending a readme for the admin..."
+
+# ynh_send_readme_to_admin --app_message="../conf/msg_install" --recipients=$admin_email --type='install'
```

Comment:
Should this instruction remain if it is commented?

Reply:
Sorry, this line was already there.

So thank you, I merge :)

Reply:
lol I haven't seen it ^^